Test.Net45/Test.Net45.csproj

```diff
@@ -36,8 +36,8 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="nunit.framework, Version=3.9.0.0, Culture=neutral, PublicKeyToken=2638cd05610744eb, processorArchitecture=MSIL">
-      <HintPath>..\packages\NUnit.3.9.0\lib\net45\nunit.framework.dll</HintPath>
+    <Reference Include="nunit.framework, Version=3.8.1.0, Culture=neutral, PublicKeyToken=2638cd05610744eb, processorArchitecture=MSIL">
```

Comment:
Curious - why did this change?

Reply:
I couldn't build test project because version 3.8.1 was referenced in [packages.config](https://github.com/segmentio/Analytics.NET/blob/master/Test.Net45/packages.config#L3). Since the same version is used across all other test projects I just fixed the path here.